login-app/src/components/App.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import { BrowserRouter as Router, Route, Switch, Link } from "react-router-dom";
+import axiosWithAuth from "../utils/api"
+import PrivateRoute from "./PrivateRoute"
+import Login from "./Login"
+import FriendsList from "./FriendsList"
+import Home from "./Home"
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <div>
+        <Route
+          path='/'
+          exact
+          render={() => (
+            <Home />
+          )}
+        />
+        <Route
+          path='/login'
+          render={props => (
+            <Login {...props} />
+          )}
+        />
+      <PrivateRoute
+          path="/protected"
```

Comment:
just a matter of convention, but I think PrivateRoute already explains to other devs that this route is protected.  The path which is consumer facing should be used to describe what the component does.